app/src/examples/PerfomanceMonitorExample.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { View } from 'react-native';
+import { PerformanceMonitor } from 'react-native-reanimated';
+
+export default function PerformanceMonitorExample() {
+  return (
+    <View>
+      <PerformanceMonitor />
+    </View>
+  );
+}
```

Comment:
We usually create examples so, that they showcase our features in action. It would be great if you added some things that the performance can be tracked on. For example, you can add some of our animations - feel free to base on any other example in the app.